src/main/java/org/apache/commons/net/util/NetConstants.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.net.util;
+
+/**
+ * <p>
+ * An constant class providing functionality related to the {@code
+ * apache.commons.net} package.
+ * </p>
+ *
+ * @since 3.7
```

Comment:
The since tag is wrong, pom.xml currently has 3.7.3 as the next version, but, since this is a new public element, we could bump the version up to 3.8.0. Let's keep 3.7.3 since this is just a constant that happens to be public for internal convenience.

Reply:
Changed